.travis.yml

```diff
@@ -22,15 +22,13 @@ env:
 
 cache: pip
 
-# command to run tests
-script:
-  - python -m pip install --editable .
-  - python -m pytest --cov=ark --pycodestyle ark
-
 jobs:
   include:
+    - stage: pytest_run
+      script:
+        - python -m pip install --editable .
+        - python -m pytest --cov=ark --pycodestyle ark
     - stage: test_pypi_deploy
-      python: 3.8
```

Comment:
Any reason we don't need 3.8 anymore?

Reply:
@ngreenwald setting `python: 3.8` at the beginning of `.travis.yml` already ensures Python 3.8 is used across all the jobs, so no need to specify it individually in each job.